src/components/Search/SearchComponent.module.scss

```diff
@@ -8,3 +8,62 @@
   margin-right: 16px;
   font-size: 17px;
 }
+
+.searchButtonWrapper {
+  display: 'flex';
+  align-items: 'center';
+
+}
+
+.searchButton {
+  cursor: pointer;
+
+}
+
+.searchContainer {
+  display: flex;
+  border: 1px solid;
+  width: 100%;
+  margin-bottom: 15px;
+  border-radius: 4px
+}
+
+.searchBox {
+  width: 100%;
+  padding: 10px;
+  border: none;
+  outline: none;
+  font-size: medium;
+}
+
+.searchBoxImg {
+  padding: 10px;
+  min-width: 40px;
+  text-align: center;
+}
+
+.resultListElement {
+  &:before{
+    content: '';
+  }
+}
+
+.resultElementCard {
```

Comment:
these boxes feel a bit out of place but we don't have our handy [`Box` component here we have over in market](https://github.com/oceanprotocol/market/blob/main/src/components/atoms/Box.module.css). It's essentially what's in this repo the [styles for each repository on front page](https://github.com/oceanprotocol/docs/blob/main/src/components/Repository/index.module.scss).

So you could copy over this `Box` style file, rename to `scss` and then use `composes: ` pattern in the (s)css modules over here, using it as shared styles for the repository & this search result box

Reply:
I have changed the styles. Do you expect results to be a grid like marketplace or current implementation is fine?

A: 
![image](https://user-images.githubusercontent.com/16358982/134775172-e606c4dd-440f-4ea1-990a-ae01ded50e47.png)

B:
![image](https://user-images.githubusercontent.com/16358982/134775191-89d638a7-ec4d-47fc-a298-d7ac50bd7c22.png)

A or B?